gulpfile.babel.js

```diff
@@ -11,7 +11,7 @@ const plugins = gulpLoadPlugins();
 const paths = {
   js: ['./**/*.js', '!dist/**', '!node_modules/**', '!coverage/**'],
   nonJs: ['./package.json', './.gitignore'],
-  tests: './server/tests/*.js'
+  tests: ['./server/tests/*.js', './server/tests/**/*.js']
```

Comment:
Does changing this to just `'./server/tests/**/*.js` do the same thing as the array?

That's what @kwm4385 did in his PR. (https://github.com/PagerNation/Escalator-API/pull/4/files)